src/actions/appActions.js

```diff
@@ -197,6 +199,8 @@ export const initAppAndRedirectAction = (appState: string, platform: string) =>
 
       await loadAndMigrate('history', dispatch, getState);
 
+      dispatch(loadBitcoinAddresses());
```

Comment:
```suggestion
      dispatch(loadBitcoinAddressesAction());
```
let's keep consistency